### PR TITLE
Add sort_values method to Series

### PR DESCRIPTION
**Pull request by floscha**
This PR adds pandas' `sort_values()` method to Koalas Series after it has already been implemented for DataFrames. While at it, I also added type hints to the DataFrames method.

### CODE REVIEW

**Comment by HyukjinKwon** [comment]
Nice. Will take a look soon

File: databricks/koalas/series.py [hunk 1]
```diff
@@ -1003,6 +1003,38 @@ def value_counts(self, normalize=False, sort=True, ascending=False, bins=None, d
...
+             the by.
+        inplace : bool, default False
+             if True, perform operation in-place
+        na_position : {'first', 'last'}, default 'last'
+             `first` puts NaNs at the beginning, `last` puts NaNs at the end
+
+        Returns
+        -------
+        sorted_obj : Series ordered by values.
+        """
```
Comment (HyukjinKwon) [inline databricks/koalas/series.py]: Eh, why don't we add the doctests as examples as well? (https://github.com/pandas-dev/pandas/blob/v0.24.2/pandas/core/series.py#L2668-L2831)

Comment (floscha) [reply databricks/koalas/series.py]: Good point. I've added them with fa32e67.

File: databricks/koalas/frame.py [hunk 2]
```diff
@@ -1861,7 +1861,8 @@ def get(self, key, default=None):
         except (KeyError, ValueError, IndexError):
             return default
 
-    def sort_values(self, by, ascending=True, inplace=False, na_position='last'):
+    def sort_values(self, by: Union[str, List[str]], ascending: Union[bool, List[bool]] = True,
+                    inplace: bool = False, na_position: str = 'last') -> Union['DataFrame', None]:
```
Comment (HyukjinKwon) [inline databricks/koalas/frame.py]: I think the return type can be`Optional['DataFrame']`

File: databricks/koalas/series.py [hunk 3]
```diff
@@ -824,6 +825,158 @@ def value_counts(self, normalize=False, sort=True, ascending=False, bins=None, d
...
+                                                   na_position=na_position))
+        if inplace:
+            self._kdf = ks_.to_dataframe()
+            self._scol = ks_._scol
+            self._index_map = ks_._index_map
+            return None
+        else:
+            return ks_
+
+    def isin(self, values):
```
Comment (HyukjinKwon) [inline databricks/koalas/series.py]: Nice, why don't we have two commits for isin and sort_values :)?

Comment (floscha) [reply databricks/koalas/series.py]: Oops, this somehow sneaked in while merging 'master' with 7637c6e :/

File: databricks/koalas/frame.py (hunk 2 above)
Comment (floscha) [reply databricks/koalas/frame.py]: Fair enough. Fixed it with 862b42f.

File: databricks/koalas/series.py (hunk 3 above)
Comment (HyukjinKwon) [reply databricks/koalas/series.py]: Ah, this was moved to `IndexOpsMixin`. Let;s remove it here.

Comment (floscha) [reply databricks/koalas/series.py]: Removed with 18249a0

**Comment by HyukjinKwon** [comment]
Merged. Thanks @floscha

**Pull request merged**